community/make-contribution/BlogSubmission.md

```diff
@@ -7,15 +7,32 @@ description: >
 
 Thank you for contributing to DevLake blog! We can't wait to hear your voice! 
 
-Because DevLake only accepts blog posts submitted through PRs, we want to make this process as painless as poissble by putting together this instruction.
+Because DevLake only accepts blog posts submitted through PRs, we want to make this process as painless as possible by putting together this instruction.
+
+## Guidelines
+
+The objective of creating blogs is to educate the community, we advise you to read the following guidelines:
+
+1. Before writing the blog, we recommend that you install DevLake and build a practical use-case.
+2. **Recommended Topics** are listed in the table below.
+
+ _[Updated on 20th March 2023]_
+ 
+| No.| Objective | Type | Example Blogs / Topics | 
+|---| ------ | --------------| ------------ |
+| 1.| Walkthrough of setup in your own toolchains | Installation | Eg: "Setting Up DevLake with GitLab and Jenkins", "Getting started with DevLake using GitHub, Argo, etc." |
+| 2.| Critical troubleshooting blogs while installing DevLake | Troubleshooting | Check out the examples from [Troubleshooting guide](https://devlake.apache.org/docs/Troubleshooting/) |
+| 3.| Your own experience of using specific feature(s) and/ or plugin(s). | Use-cases | E.g. "How to use PagerDuty", "GitLab and CircleCI to measure DORA metrics", "How to use team feature in Apache DevLake", "Creating your own dev-metrics with DevLake". | 
```

Comment:
Text change:
E.g. "How to use Jira, GitLab and Jenkins to measure DORA metrics", "How to get the team configured in Apache DevLake", "How to create your own dev-metrics with DevLake".